client/angular/components/group_page/group_actions_dropdown/group_actions_dropdown.coffee

```diff
@@ -11,6 +11,18 @@ angular.module('loomioApp').directive 'groupActionsDropdown', ->
   replace: true
   controller: ['$scope', ($scope) ->
 
+    $scope.canExportData = ->
+      Session.user().isMemberOf($scope.group)
+
+    $scope.openGroupExportModal = ->
+      ModalService.open 'ConfirmModal', confirm: ->
+        submit: -> Records.groups.export($scope.group.id)
```

Comment:
We tend towards actions on the model, so
```
# group_model.coffee
export: =>
  @remote.post(@id, 'export')
```
```
submit: $scope.group.export
```

Reply:
done, thanks